src/templates/landing_pages/ico.html.tt

```diff
@@ -662,8 +663,26 @@
                     </div>
                     <div class="gr-12">
                         <div class="faq">
+                            <h4>[% l("How many tokens are available?") %]</h4>
+                            <p>[% l("Up to 10 million tokens are available via the auction.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
+                            <h4>[% l("What is the minimum bid price?") %]</h4>
+                            <p>[% l("The minimum bid is USD 1.35/token or equivalent in other currency/cryptocurrency.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
+                            <h4>[% l("How high should I bid?") %]</h4>
+                            <p>[% l("The higher you bid, the more your chances of being successful in the auction. However irrespective of how high you bid, at the end of the auction all winning bids will receive the same final price, which is the lowest winning bid.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
                             <h4>[% l("Who can purchase the tokens?") %]</h4>
-                            <p>[% l("Tokens will only be offered to [_1] clients who have a Real Money account, and who have completed our [_2]account authentication/KYC process[_3].", broker_name, '<a href="https://ico_documents.binary.com/2017_Authentication_Process.pdf" target="_blank">', '</a>') %]</p>
+                            <p>[% l("Tokens will only be offered to [_1] clients who have a Real Money account. Please visit <a href='[_1]' target='_blank'>www.binary.com</a> to open an account.", broker_name, 'www.binary.com') %]</p>
```

Comment:
link isn't working - do you need to add https:// to the URL?

Reply:
i've fixed the link. need to `request.url_for('home')`